### PR TITLE
doc: add tip on extracting binary release

### PR DESCRIPTION
**Pull request by maxwedwards**
How and where to extract the binaries has come up on Discord at least a couple of times. Hopefully this tip will help.

### CODE REVIEW

**Review by vincenzopalazzo — APPROVED** [review]
Make sense!

ACK https://github.com/ElementsProject/lightning/pull/6633/commits/a7e0c2d6de3cf6b6f9fcc2e9173b97a360c3ba0e

File: doc/getting-started/getting-started/installation.md [hunk 1]
```diff
@@ -17,7 +17,11 @@ sudo snap install bitcoin-core
 sudo ln -s /snap/bitcoin-core/current/bin/bitcoin{d,-cli} /usr/local/bin/
 ```
 
-Then you can fetch a pre-compiled binary from the [releases](https://github.com/ElementsProject/lightning/releases) page on GitHub. Core Lightning provides binaries for both Ubuntu and Fedora distributions.
+Then you can fetch a pre-compiled binary from the [releases](https://github.com/ElementsProject/lightning/releases) page on GitHub. Core Lightning provides binaries for both Ubuntu and Fedora distributions. Normally these binaries are extracted into /usr/local:
+
+```shell
+sudo tar -xvf <release>.tar.xz -C /usr/local --strip-components=2
```
Comment (rustyrussell) [inline doc/getting-started/getting-started/installation.md:23]:
```
sudo tar -xvaf <release>.tar.xz -C /usr/local --strip-components=2
```

Without the `a` it won't work...

**Review by rustyrussell — CHANGES_REQUESTED** [review]

File: doc/getting-started/getting-started/installation.md (hunk 1 above)
Comment (rustyrussell) [reply doc/getting-started/getting-started/installation.md:23]: Wait, it does... what wild magic is this?!!

Comment (rustyrussell) [reply doc/getting-started/getting-started/installation.md:23]: TIL!  Man page is silent, but info page:

```
   Reading compressed archive is even simpler: you don't need to specify
any additional options as GNU 'tar' recognizes its format automatically.
Thus, the following commands will list and extract the archive created
in previous example:

     # List the compressed archive
     $ tar tf archive.tar.gz
     # Extract the compressed archive
     $ tar xf archive.tar.gz

   The format recognition algorithm is based on "signatures", a special
byte sequences in the beginning of file, that are specific for certain
compression formats.  If this approach fails, 'tar' falls back to using
archive name suffix to determine its format (*note auto-compress::, for
a list of recognized suffixes).
```

**Review by rustyrussell — APPROVED** [review]

**Pull request merged**